ditto/modify/modify.py

```diff
@@ -37,7 +37,12 @@ def copy(self, model, obj):
                     getattr(obj, attr)
                 ) == 0 or attr == 'reactances' or attr == 'phases' or attr == 'impedance_matrix' or attr == 'capacitance_matrix': #Reactances (PowerTransformer) and phases (Node) are a special case of lists that aren't classes
                     new_attr = getattr(obj, attr)
-                    setattr(new_obj, attr, new_attr)
+                    if type(new_attr).__name__ == 'list':
+                        setattr(new_obj,attr,[])
+                        for value in new_attr:
+                            getattr(new_obj,attr).append(value)
+                    else:
+                        setattr(new_obj, attr, new_attr)
```

Comment:
I think that #63 solved the issue we were having with this. So you should be able to set the attribute with the full list rather than creating an empty list and adding elements to it. That is the previous 

```python
setattr(new_obj, attr, new_attr)
```
should work.
 I think it matters because when appending elements, there is no type checking (see #51).